doc/languages-frameworks/qt.section.md

```diff
@@ -0,0 +1,113 @@
+
+Qt
+=
+This section describes the differences between Nix expressions for Qt libraries and applications and Nix expressions for other C++ software. Some knowledge of the latter is assumed.
+
+There are primarily two problems which the Qt infrastructure is designed to address: ensuring consistent versioning of all dependencies and finding dependencies at runtime.
+
+**Nix expression for a Qt package (default.nix)**
```

Comment:
Please use actual headings, for headings, not bold paragraphs.

Reply:
Have updated this.